spec/lib/logging/third_party_transaction_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Logging::ThirdPartyTransaction do
```

Comment:
This file also lost some git history when moving.  Notable changes are:

1. location and namespace
2. refactoring from a single set of specs to a shared example group
3. addition of return value verification spec